src/index.js

```diff
@@ -54,4 +54,45 @@ document.querySelector('#task-list').addEventListener('change', (e) => {
     // Reload page
     location.reload();
   }
+});
+
+// Event: editing and removing
+document.querySelector('#task-list').addEventListener('click', (e) => {
+  const classesIcn = e.target.className;
+  const classesArr = classesIcn.split(' ');
+
+  const li = e.target.parentElement.parentElement;
+  const ulList = document.querySelector('#task-list');
+  const nodes = Array.from(ulList.children);
+  const index = nodes.indexOf(li);
+
+  // when the three dots icon gets clicked
+  if (classesArr.indexOf('edtIcn') !== -1) {
+    UI.changeLiToEditMode(li);
+  }
+
+  // when the check icon gets clicked to UPDATE
+  if (classesArr.indexOf('removeIcn') !== -1) {
+    UI.removeTask(index);
+
+    // Reload page
+    location.reload();
+  }
+
+  // when the the trash icon gets clicked to REMOVE
+  if (classesArr.indexOf('acceptIcn') !== -1) {
+    const newDesc = document.querySelector('#inputEdit').value;
+    UI.updateTask(index, newDesc);
+
+    // Reload page
+    location.reload();
+  }
+});
+
+document.querySelector('#pBtm').addEventListener('click', () => {
+  console.log('hi');
```

Comment:
- [ ] Please remove all console outputs. Please check [the JavaScript best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for more information.